website/static/css/timeline.css

```diff
@@ -0,0 +1,51 @@
+.timeline-container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.timeline-event {
+  margin: 0;
+
+  position: relative;
+  display: flex;
+
+  text-align: left;
+}
+
+.timeline-event::before {
+  content: "";
+  background-color: #ffd533;
+  position: absolute;
+  bottom: 0;
+  top: 0;
+  left: 1.5rem;
+  width: 3px;
+}
+
+.timeline-event-dot {
+  min-width: 3rem;
+  width: 3rem;
+  height: 3rem;
+  margin: 0;
+  margin-right: 1em;
+  margin-top: 1rem;
+
+  background: #ffd533;
+  border-radius: 50%;
+  font-weight: bold;
+  font-size: 0.8em;
+
```

Comment:
I feel like the bold black text is competing with the section title. I tried to inverse the color in timeline dot:

```css
hsla(0,0%,100%,.9)
```

![image](https://user-images.githubusercontent.com/3607926/114572951-f01c6000-9c45-11eb-98ee-72254803c924.png)